docs/source/user-guide/concepts.rst

```diff
@@ -6,6 +6,48 @@ Concepts
    :local:
    :depth: 1
 
+.. image:: /img/installing-with-conda.png
+    :align: right
+
+.. _installing-with-conda:
+
+Installing with conda
+=====================
+
+Conda install
+-------------
+* When you `conda install` a package that exists in your channels and has no dependencies, conda:
```

Comment:
"your" seems a bit odd here as the majority of users will be installing from a channel they are not owners of.  Perhaps "a channel" would be better.